packages/client/utils/AzureDevOpsClientManager.ts

```diff
@@ -40,8 +40,8 @@ class AzureDevOpsClientManager {
     const providerState = Math.random().toString(36).substring(5)
     const verifier = AzureDevOpsClientManager.generateVerifier()
     const code = await AzureDevOpsClientManager.generateCodeChallenge(verifier)
-    const redirect = makeHref('/auth/ado')
-    const scope = '499b84ac-1321-427f-aa17-267ca6975798/.default'
+    const redirect = makeHref('/auth/ado2')
```

Comment:
I switched our app to be a "web" app rather than "spa" so our access token gets a longer life-span. Because the redirects are not compatible, I just appended the 2 for now so it works straight after deploying.